### PR TITLE
Add missing needles for yast2_ntpclient test on Leap15

### PR DESCRIPTION
**Pull request by ldevulder**
yast2_ntpclient test as been improved to support chronyd on
SLE15/Leap15, so new needles are need for openSUSE.

Failing test with missing needles: https://openqa.opensuse.org/tests/584343#step/yast2_ntpclient/9

### CODE REVIEW

File: yast2_ntp-client_support_server-bsc1074726-20180104.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "xpos": 303,
+      "ypos": 366,
+      "width": 410,
+      "height": 19,
+      "type": "match"
+    }
+  ],
+  "tags": [
+    "bsc#1074726"
```
Comment (okurz) [inline yast2_ntp-client_support_server-bsc1074726-20180104.json]: If this is a workaround needle for a bug then it should have the according property

File: yast2_ntpclient-20180103.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    {
+      "height": 36,
+      "width": 227,
+      "ypos": 300,
+      "type": "match",
+      "xpos": 38
+    }
+  ],
+  "tags": [
+    "ENV-SP2ORLATER-1",
```
Comment (okurz) [inline yast2_ntpclient-20180103.json]: Delete, doesn't apply here

Comment (ldevulder) [reply yast2_ntpclient-20180103.json]: Yes, I will remove it.

File: yast2_ntp-client_support_server-bsc1074726-20180104.json (hunk 1 above)
Comment (ldevulder) [reply yast2_ntp-client_support_server-bsc1074726-20180104.json]: What property? I don't understand :)
I use this tag in the `yast2_ntpclient` code. Or do I need an boo# number nstead of a bsc#?

Comment (okurz) [reply yast2_ntp-client_support_server-bsc1074726-20180104.json]: see https://progress.opensuse.org/projects/openqatests/wiki/#applying-quotworkaroundquot-needles . In the needle editor you can click the "workaround" checkbox which will fill an entry to "properties". Your needle name is fine. bsc# is fine. It most likely will not be visible to non-SUSE users anyway.

Comment (ldevulder) [reply yast2_ntp-client_support_server-bsc1074726-20180104.json]: OK, I will try to figure out how to do this :)

File: yast2_ntp-client_support_server-bsc1074726-20180104.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "properties": [],
```
Comment (okurz) [inline yast2_ntp-client_support_server-bsc1074726-20180104.json]: still, we need a property workaround here. See other needle json files as example

Comment (ldevulder) [reply yast2_ntp-client_support_server-bsc1074726-20180104.json]: Changed!

Do I need to do the same thing for the already accepted needle for SLE?

**Pull request merged**